test/functional/tests/ioctl/test_ioctl_start.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from time import sleep
+
+from api.cas.casadm import start_cache
+from api.cas.casadm_parser import get_caches
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from test_tools.cas_ioctl.cas_requests import StartCacheRequest
+from test_tools.cas_ioctl.cas_structs import CacheMode, CacheLineSize, InitCache
+from test_tools.cas_ioctl.ioctl import cas_ioctl
+from test_utils.size import Size, Unit
+from tests.ioctl import common_utils
+
+cache_id = 3
+
+
+@pytest.mark.parametrizex("force", [0, 1])
+@pytest.mark.parametrizex("load", InitCache)
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_ioctl_start(cache_mode, cache_line_size, load, force):
+    """
+        title: Start the cache without casadm.
+        description: |
+          Test of the ability to start the cache with IOCTL request bypassing native
+          OpenCAS manager - casadm.
+        pass_criteria:
+          - Cache started successfully without any errors.
+          - When 'force' and 'load' flags are used cache is not started.
```

Comment:
The first bullet is in the opposite to the second one. Shouldn't all the pass criteria be fulfilled?